Ginger/GingerCoreNET/Reports/BusinessFlowReport.cs

```diff
@@ -384,6 +385,14 @@ public int TotalActivitiesOther
             }
         }
 
+        [FieldParams]
+        [FieldParamsNameCaption("Business Flow Activities Execution Rate")]
```

Comment:
Remove the 'Activities' from "Business Flow Activities Execution Rate" because this is relevant only to legacy report, in liteDB it might be based on Actions